### ISSUE TITLE
2.7.0 is missing files

### ISSUE THREAD

**Issue by tneems** [evanphx/benchmark-ips#74]
```ruby
2.3.1 :001 > require 'benchmark/ips'
LoadError: cannot load such file -- benchmark/ips/stats/sd
	from /Users/tneems/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/tneems/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/tneems/.rvm/gems/ruby-2.3.1/gems/benchmark-ips-2.7.0/lib/benchmark/ips.rb:4:in `<top (required)>'
	from /Users/tneems/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:133:in `require'
	from /Users/tneems/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:133:in `rescue in require'
	from /Users/tneems/.rvm/rubies/ruby-2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:40:in `require'
	from (irb):1
	from /Users/tneems/.rvm/rubies/ruby-2.3.1/bin/irb:11:in `<main>'
```

**Comment by matthewrudy** [comment]
Just hit this too
thanks for the fix @tneems

**Comment by jonatack** [comment]
This issue was fixed last Saturday with #72.
